astropy/io/registry.py

```diff
@@ -318,12 +320,20 @@ def register_identifier(data_format, data_class, identifier, force=False):
         _identifiers[(data_format, data_class)] = identifier
     else:
         raise IORegistryError("Identifier for format '{0}' and class '{1}' is "
-                              'already defined'.format(data_format,
-                                                       data_class.__name__))
+                              'already defined.'.format(data_format,
+                                                        data_class.__name__))
 
 
 def identify_format(origin, data_class_required, path, fileobj, args, kwargs):
-    # Loop through identifiers to see which formats match
+    """Loop through identifiers to see which formats match.
+
+    For the parameter description see `register_identifier`.
```

Comment:
Yes, I made the mistake others might make, which is assuming you mean the parameters for `identify_format` are literally the same as for `register_identifier`, which is not the case.  The right answer is that they are similar to the sub-parameters for the `identifier` parameter within `register_identifier`.

At the end of the day doing docs by reference is not the typical pattern, even if it has the slight advantage of ensuring consistency.